src/prototype_room_my.js

```diff
@@ -488,30 +482,9 @@ Room.prototype.executeRoom = function() {
   this.handleTerminal();
   this.handleNukeAttack();
 
-  var creep;
-  var creep_index;
-
-  for (var creep_name in creepsInRoom) {
-    creep = creepsInRoom[creep_name];
-    creep_index = creepsConfig.indexOf(creep.memory.role);
-    if (creep_index != -1) {
-      creepsConfig.splice(creep_index, 1);
-    }
-  }
-
-  for (var spawn_name in spawns) {
-    spawn = spawns[spawn_name];
-    if (!spawn.spawning || spawn.spawning === null) {
-      continue;
-    }
-
-    creep = Game.creeps[spawn.spawning.name];
-    creep_index = creepsConfig.indexOf(creep.memory.role);
-    if (creep_index != -1) {
-      creepsConfig.splice(creep_index, 1);
-    }
+  if (Game.time % 10 === 0) {
+    this.spawnCheckForCreate();
```

Comment:
Totally removed old creepsConfig system. @TooAngel good if you can quickly recheck :)